enterprise/backend/src/metabase_enterprise/metabot_v3/dummy_tools.clj

```diff
@@ -100,6 +107,31 @@
       (get-table-details :get-table-details {:table_id id} {})))
   -)
 
+(defn- get-metric-details
+  [_ {:keys [metric_id]} context]
+  (let [details (card-details metric_id)
+        date-field-id (:id (first (filter #(= (:type %) "date") (:fields details))))
+        details' (some-> details
+                         (assoc :queryable_dimensions (map convert-field (visible-columns (:dataset_query (t2/select-one :model/Card metric_id))))
+                                :default_time_dimension_field_id date-field-id)
+                         (select-keys [:id :description :name :default_time_dimension_field_id :queryable_dimensions]))]
+    {:output (or details' "metric not found")
+     :context context}))
```

Comment:
I have an alternative implementation of this in #51315. ATM I think that the other `get-...-details` tools should be re-written to that style too for uniformity, if we the field IDs generated there are good enough for the LLM.